setup.py

```diff
@@ -39,7 +39,7 @@ def read(*names, **kwargs):
 
 setup(
     name='raiden-services',
-    version='0.1.0',
```

Comment:
Did the version go down here???

Reply:
Yes, we never released before and I just wanted to reserve the PyPI package name